016_IP_L151/016_IP_L151_safe.py

```
@@ -0,0 +1,366 @@
+        listk=[a]
+        for i in range(len(n)):
+            lists=[n[i],l[i]]
+            listk.append(lists)
+        bird_list.append(listk)
+    with open("l.txt",'w',encoding='utf-8') as f:
+        f.write(str(bird_list))
+    return render_template('sheet.html',bird=bird_list)  
+@app.route('/imform')  
+def imform_net():  
+    page=request.args.get("page")
+    if page == None or page=='' or page == ' ':
```

Ruff (0.14.1)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)